CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## next
 
+*Features*
```

Comment:
I think this needs to be rebased or it might incorrectly get merged into the 0.25.0 section